.storybook/theo-importer.js

```diff
@@ -0,0 +1,42 @@
+const { dirname, resolve } = require('path');
+const theo = require('theo');
+
+/**
+ * Import Theo YAML files into SCSS files.
+ *
+ * @param {string} url - File path relative to SCSS file.
+ * @param {string} prev - File path of SCSS file that made import request.
+ * @param {function({ file: string, contents: string })} done - Callback to
+ * pass results to.
+ * @link <https://basalt.io/blog/theo-design-tokens-using-node-sass-importer-for-any-build-method>
+ * @link <https://github.com/sass/node-sass#importer--v200---experimental>
+ */
+function theoImporter(url, prev, done) {
```

Comment:
`url` and `prev` are somewhat confusing variable names. I'd suggest `yamlPath` and `scssPath` to make things a bit more clear.

Reply:
@spaceninja I thought the same, but these are actually the variable names defined in Sass's importer documentation. Keeping these the same made it a bit easier to troubleshoot by referring to the Sass API docs.

Reply:
👍  that makes sense

Reply:
> I thought the same, but these are actually the variable names defined in Sass's importer documentation. Keeping these the same made it a bit easier to troubleshoot by referring to the Sass API docs.

Can a comment be added explaining this so a future dev doesn't have the same question/thought? 🙂

Reply:
@gerardo-rodriguez Done!